spark/src/main/antlr/FlintSparkSqlExtensions.g4

```diff
@@ -26,6 +26,7 @@ skippingIndexStatement
     | refreshSkippingIndexStatement
     | describeSkippingIndexStatement
     | dropSkippingIndexStatement
+    | vacuumSkippingIndexStatement
```

Comment:
Why do we need to update .g4?

Reply:
These g4 files come from opensearch-spark. We always keep them up to date.